module/cpf.sh

```diff
@@ -26,17 +29,22 @@ declare -A COLORS=(
 )
 
 #. cpf:module -={
-function ::cpf:module_is_modified() {
+function ::cpf:module_is_unchanged() {
```

Comment:
I cleaned this up and pushed - have a look and let me know if it makes sense to you?